pandas/io/parsers.py

```diff
@@ -270,43 +259,63 @@
 encoding : str, default None
     Encoding to use for UTF when reading/writing (ex. 'utf-8'). `List of Python
     standard encodings
-    <https://docs.python.org/3/library/codecs.html#standard-encodings>`_
+    <https://docs.python.org/3/library/codecs.html#standard-encodings>`_ .
 dialect : str or csv.Dialect instance, default None
     If provided, this parameter will override values (default or not) for the
     following parameters: `delimiter`, `doublequote`, `escapechar`,
     `skipinitialspace`, `quotechar`, and `quoting`. If it is necessary to
     override values, a ParserWarning will be issued. See csv.Dialect
     documentation for more details.
-tupleize_cols : boolean, default False
+tupleize_cols : bool, default False
+    Leave a list of tuples on columns as is (default is to convert to
+    a MultiIndex on the columns).
     .. deprecated:: 0.21.0
```

Comment:
Did you check if this renders well? If I remember correctly, this should have a blank line before.